test/src/e2e.long/tendermint.test.ts

```diff
@@ -255,9 +255,9 @@ describe("Tendermint ", function() {
         await promiseExpect.shouldFulfill(
             "wait blocknumber",
             Promise.all([
-                nodes[0].waitBlockNumber(startHeight + 1),
```

Comment:
What is the purpose of this test?
Does it test the block sync extension works?
If then, I think it should be changed to the below scenario.
1. Connect three nodes.
2. Generate blocks.
3. Connect one more node.
4. Check whether the last node follows the synchronization.

Reply:
I want to check two things.

1. Whether the block extension is working or not working.
2. Whether the node which received blocks using block extension could participate or could not participate in the Tendermint consensus.